src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -1,5 +1,22 @@
 package org.stellar.walletsdk
 
+import org.stellar.sdk.KeyPair
+import org.stellar.sdk.Network
+import org.stellar.sdk.Server
+import shadow.com.google.gson.annotations.SerializedName
+
 class Wallet(private val horizonUrl: String, private val horizonPassphrase: String) {
-    // TODO: Wallet code goes here
-}
\ No newline at end of file
+  private val server = Server(this.horizonUrl)
+  private val network = Network(this.horizonPassphrase)
+
+  data class AccountKeypair(
+    @SerializedName("publicKey") val publicKey: String,
```

Comment:
Do you use `gson` or plan to serialize this class to JSON format? If not, I think we may not need `@SerializedName` annotation.

Reply:
@lijamie98 What would you recommend? For web, we would definitely use JSON, but I'm not sure if that's the best format for Kotlin.

Reply:
@lijamie98 Is this the right way to define return type in this case (in `create()` method)? :pray:

Reply:
Java has two popular libraries `gson` and `jackson` to convert an object to JSON string. In AP, we use `gson`. 

`@SerializedName` is only used when 
1. You need to send the object in JSON notation. Otherwise, you don't have to annotate it.
2. [When to serialize a field that has different name from the Java field name](https://www.tutorialspoint.com/what-to-use-serializedname-annotation-using-gson-in-java#:~:text=The%20%40SerializedName%20annotation%20can%20be,field%20with%20the%20provided%20name.). 

In your case, your Java field name equals to the JSON field name. So that you do not have to annotate it. :-)

Reply:
Thanks for the detailed explanation, Jamie! We don't need JSON here, so I removed `@SerializedName`. :rocket: